### PR TITLE
Feat/cc 146 Intro section landing page

### PR DESCRIPTION
**Pull request by wwwseven**
- Create responsive intro section
- Add slide in animation
- Add on scroll effect

intro

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+  .btn-primary {
+    @apply bg-pink px-6 py-1.5 rounded-md font-semibold text-sm text-btnColor;
+  }
+}
```
Comment (tatoMa) [inline styles/globals.css]: Nice one.

File: components/Intro/index.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        alt="basketball court"
+        width={528}
+        height={402}
+        className={`${inView && "animate-introToRight1"}`}
+      />
+      <TextBox
+        inView={inView}
+        title="basketball"
+        text="Design your dream basketball.Design your dream basketball.Design your dream basketball.Design your dream basketball.Design your dream basketball."
+      />
```
Comment (tatoMa) [inline components/Intro/index.tsx]: Please update all the text based on the picture BA provided.

**Comment by tatoMa** [comment]
I don't know if it is just me. The animation in this section looks a little weird on my computer.

File: components/Intro/TextBox.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  return (
+    <div
+      className={`md:max-w-[22rem] self-center md:justify-self-start ${
+        inView && (isTennis ? "animate-introToRight2" : "animate-introToLeft1")
+      }`}
+    >
+      <h2 className="text-h2 mb-2">Design your dream {title} court</h2>
+      <p className="text-p opacity-85 mb-4">{text}</p>
+      <button className="btn-primary w-full md:w-fit">Get Started</button>
```
Comment (tatoMa) [inline components/Intro/TextBox.tsx]: Link of this button?

File: components/Intro/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import Image from "next/image";
+import TextBox from "./TextBox";
+import { useInView } from "react-intersection-observer";
+
+const Intro = () => {
+  const { ref, inView } = useInView();
+
+  return (
+    <section
+      className="max-w-7xl mx-auto px-3 sm:px-8 lg:px-20 py-12 bg-black/30 overflow-hidden text-white grid grid-cols-1 md:grid-cols-2 gap-x-16 gap-y-6 justify-items-center"
```
Comment (Double-Twelve) [inline components/Intro/index.tsx]: 1. in mobile, the row gap is incorrect
2. should remove `justify-items-center`

File: components/Intro/TextBox.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const TextBox = ({ title, text, inView }: any) => {
+  const isTennis = title === "tennis";
+
+  return (
+    <div
+      className={`md:max-w-[22rem] self-center md:justify-self-start ${
```
Comment (Double-Twelve) [inline components/Intro/TextBox.tsx]: should remove `md:justify-self-start`

File: components/Intro/TextBox.tsx (hunk 3 above)
Comment (Double-Twelve) [inline components/Intro/TextBox.tsx]: line height is incorrect

File: components/Intro/TextBox.tsx (hunk 5 above)
Comment (wwwseven) [reply components/Intro/TextBox.tsx]: “justify” is set for special screen size

**Review by tatoMa — APPROVED** [review]

**Pull request merged**